src/elm/Data/Inventory.elm

```diff
@@ -194,6 +202,15 @@ purchaseResource count level (Inventory ({ resources, wallet } as inventory)) =
                 Inventory inventory
 
 
+applyDiscounts : Resource.Level -> LimitedMultipliers.Model -> Currency.Currency -> Currency.Currency
```

Comment:
Thoughts on `applyPurchaseDiscounts` or something a bit more targeted to the notion that it's a discount applied during purchase?